lib_dsp/src/dsp_sqrt_xs2.S

```diff
@@ -16,7 +16,7 @@
     .align 16
     .globl sqrt_as
     .issue_mode dual
-dsp_sqrt30_xs2:
+dsp_sqrt30_xs2:

```

Comment:
Somehow this file has ended up with this single line having a unix file ending - so this change fixes that so all lines in the file have a dos ending.